client/src/main/js/factory/Reserva.js

```diff
@@ -110,6 +110,10 @@ let sequenceReserva = 1;
             return `${this.inicio}-${this.fim}`;
         };
 
+        Reserva.prototype.ehReservaPassada = function () {
+            return this.end < Date.now();
```

Comment:
É `this.end` msm? Isso não deveria estar em português?

Reply:
Tá em inglês porque o calendário precisa das propriedades 'start' e 'end' pra definir começo e fim dos eventos.

Reply:
Mas vc pode usar o `this.fim` aí